modules/sdk-core/src/account-lib/baseCoin/enum.ts

```diff
@@ -48,6 +48,8 @@ export enum TransactionType {
   // Cross chain transfer at Avax
   Export,
   Import,
+  // Cross chain Recovery for evm like chains
+  RecoveryWalletDeployment,
```

Comment:
CrossChainRecovery is more meaningFull?

Reply:
I think TransactionType can help to indicate the type of transaction like send,deployment etc.
so accordingly I feel `crossChainRecoveryWalletDeployment` is a bit complex